hudi-hadoop-mr/src/test/java/org/apache/hudi/hadoop/realtime/TestHoodieRealtimeFileSplit.java

```diff
@@ -86,7 +78,7 @@ public void testWrite() throws IOException {
 
     // register expected method calls for void functions
     // so that we can verify what was called after the method call finishes
-    doNothing().when(out).writeByte(anyByte());
+    doNothing().when(out).writeByte(anyInt());
```

Comment:
writeByte() actually takes an int.. so strict stubbing requires an int for this.